lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
         for filetype, ext in sorted(FigureCanvasWebAggCore.
                                     get_supported_filetypes_grouped().
                                     items()):
-            extensions.append(ext[0])
+            if not ext[0] == 'pgf':  # pgf does not support BytesIO
```

Comment:
Backporting is fine, but what I was getting at is that it's probably a bug that `pgf` doesn't support BytesIO, and we should fix that rather than just not allowing it in the `NbAgg` interface.  Though this is fine for now, we'll probably want to undo it later.